redasql/result_formatter.py

```diff
@@ -81,9 +81,11 @@ def _format_result_to_row_base(self):
 
     def _format_result_to_column_base(self):
         row_for_tables = []
-        for row in self.rows:
+        for i, row in enumerate(self.rows, 1):
             for column_name in self.column_name_list:
                 row_for_tables.append([column_name, self._replace_line_break(row[column_name])])
+            if i < len(self.rows):
```

Comment:
Is this process trying to determine the last loop?
If it's true,  implement `signal_last` in `utils.py`
i.e. https://betterprogramming.pub/is-this-the-last-element-of-my-python-for-loop-784f5ff90bb5


```python
from typing import Iterable, Any, Tuple
def signal_last(it:Iterable[Any]) -> Iterable[Tuple[bool, Any]]:
    iterable = iter(it)
    ret_var = next(iterable)
    for val in iterable:
        yield False, ret_var
        ret_var = val
    yield True, ret_var
```

Reply:
I see! So that's the way to do it. Very cool.
I fixed it.

[ebdc898](https://github.com/denzow/redasql/pull/38/commits/ebdc8987fefd0f61ac7ac03f872e125a3afdead0)

I do not want to show the separator in the last row, so we use signal_last in two loop.